src/backend/marsha/bbb/tests/test_api.py

```diff
@@ -940,3 +1103,363 @@ def test_api_bbb_end_classroom_instructor(self, mock_end_request):
             },
             response.data,
         )
+
+
+@override_settings(BBB_ENABLED=True)
+class ClassroomDocumentAPITest(TestCase):
+    """Test for the ClassroomDocument API."""
+
+    maxDiff = None
+
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+
+        # Force URLs reload to use DEPOSIT_ENABLED
+        reload_urlconf()
+
+    def test_api_deposited_file_create_student(self):
```

Comment:
In all your methods in this class you are using `deposited_files`

Reply:
<img src="https://media0.giphy.com/media/HufOeXwDOInlK/giphy.gif"/>